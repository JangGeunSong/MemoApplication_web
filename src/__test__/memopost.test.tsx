import React from 'react'
import { render } from '@testing-library/react'
import Memopost from '../components/memo-post'

describe('memo-post component test', () => {
    const uri = 'http://localhost:5000/memos'

    it('render post component', async () => {
        const sendToModal = () => {console.log("hello")}
        const { container } = render(<Memopost 
            key={0}
            idx={0}
            title={'title1'} 
            description={'hello'} 
            background={"#fff"} 
            onClick={sendToModal}
        />)

        expect(container).toMatchSnapshot()
    })
})
