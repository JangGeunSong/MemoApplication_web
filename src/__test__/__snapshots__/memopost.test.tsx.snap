// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`memo-post component test render post component 1`] = `
<div>
  <div
    class="post-it"
    style="background: rgb(255, 255, 255);"
  >
    <h1
      class="post-title"
    >
      title1
    </h1>
    <hr
      class="post-line"
    />
    <p
      class="post-description"
    >
      hello
    </p>
  </div>
</div>
`;
